setup.py

```diff
@@ -32,9 +32,7 @@ def _find_msbuild_tool(tool="msbuild.exe", use_windows_sdk=False):
     if use_windows_sdk:
         sdks_root = r"SOFTWARE\Microsoft\Microsoft SDKs\Windows"
         kits_root = r"SOFTWARE\Microsoft\Windows Kits\Installed Roots"
-        kits_suffix = "bin"
-        if PLATFORM == "x64":
-            kits_suffix += r"\x64"
+        kits_suffix = os.path.join("bin",PLATFORM)
```

Comment:
note that #194 is linked with other pull requests. @tonyroberts is going to merge these changes once appveyor CI gets fixed.